_posts/2022-05-04-RTK-Intership.md

```diff
@@ -0,0 +1,249 @@
+{: refdef}
+
+### Testing of the antennas
+I tested both antennas side by side for 5 minutes while outside on the table, using SBAS as the correction method.
+These are the results (the more compact the data points the more accurate):
+
+{:refdef: style="text-align: center;"}
+<img src="{{ '/img/2022-05-04-RTK-Internship/TestingAntennasOutside.JPG' | prepend: site.baseurl }}" alt="TestingAntennasOutside" class="image center" style="margin:0px auto; max-width:100%">
+{: refdef}
+
+
+u-blox ANN-MB-00-00 (new antenne):
+{:refdef: style="text-align: center;"}
+<img src="{{ '/img/2022-05-04-RTK-Internship/newantenna.png' | prepend: site.baseurl }}" alt="newantenna" class="image center" style="margin:0px auto; max-width:100%">
+{: refdef}
+
+You can see that the antenna stays within a range of max 2.5m when stationary. 
+It got a connection to the signal immediately when I started the module.
+
+Unknown first cheap antenna:
+{:refdef: style="text-align: center;"}
+<img src="{{ '/img/2022-05-04-RTK-Internship/Cheapantenna.png' | prepend: site.baseurl }}" alt="Cheapantenna" class="image center" style="margin:0px auto; max-width:100%">
+{: refdef}
+
+First of all it took way longer to establish a connection, a couple of minutes. 
+The data points are further apart from each other. 
+Whereas you can see with the other antenna that they are much closer together. 
+So that means that there is a lot of play between each data point. 
+This means that the new antenna is more accurate. 
+At last you can also see that the range of the antenna went over the 2.5 meter mark.
+
+
+From this we can conclude that it is necessary to have a good antenna when we want to work precisely.
+
+### Testing with RTK
+There were a couple of problems with the SparkFun GPS-RTK Board during my internship, it didn't calculate the accurate positioning altho it did receive the correction data correctly from FLEPOS.
+The Sparkfun RTK-board is supposed to calculate this automatically, but it didn't and I could not change the configuration to the right settings.
+That is why I don't have a video which shows the use of RTK on the SparkFun GPS-RTK Board like in the section with SBAS.
+
+
+## Sending data to the cloud
+As an extra you can send the data to the cloud, during my internship at Ordina I did this as follows:
+- I received the positioning data with a Raspberry Pi and sent it to the Azure IoT Hub
+- Then through Azure functions the data got stored in a Mongodb with Azure Cosmos DB-API
+- After that, the data got handled with springboot and visualized.
+
+In the image below you can see the diagram that visualizes this process:
+{:refdef: style="text-align: center;"}
+<img src="{{ '/img/2022-05-04-RTK-Internship/DiagramRTK.png' | prepend: site.baseurl }}" alt="DiagramRTK" class="image center" style="margin:0px auto; max-width:100%">
+{: refdef}
+
+## Conclusion
+Real-Time Kinematic (RTK) can be very accurate and handy in some situations where needed.
```

Comment:
The conclusion is a bit small / too little conclusion. Explain in a few sentences what you found (RTK is acurate to x SBAS is accurate to y, small antenna vs good antenna, support for local systems is high)